presto-main/src/main/java/com/facebook/presto/operator/AriaLookupSource.java

```diff
@@ -0,0 +1,153 @@
+package com.facebook.presto.operator;
+
+import com.facebook.presto.spi.Page;
+import com.facebook.presto.spi.PageBuilder;
+import io.airlift.slice.Slice;
+
+import java.util.Arrays;
+
+import static io.airlift.slice.UnsafeSlice.getLongUnchecked;
+
+public class AriaLookupSource
+            implements LookupSource
+{
+    int statusMask;
```

Comment:
This, and everything that depends on it, will be cleaned up in an upcoming commit